### DISCUSSION TITLE
Welcome to Stardock-Software-Localization Discussions!

### DISCUSSION THREAD

**Discussion by magnumpci** [StardockCorp/Stardock-Software-Localization#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dejan-G** [comment]
When do you plane to start translation project for Start11 v2? I would like to prepare translation for ver. 2 but there is only english file (en.lng) for ver. 1.47.

Br, D

**Reply by magnumpci** [reply]
A base en.lng file has not been created yet (its still v1).  I will update the Start11 readme.md when its been updated and pushed here.

When it is available, it will also have preliminary translations for all the supported languages (FR included). Per how we are handling this, it is then where we will solicit help to make sure they read well \ fit fully in the Start11 v2 UI.

We really do appreciate the willingness to help here, Dejan.

Sean Drohan
Stardock Product Lifecycle Manager

**Comment by FJMatad** [comment]
Hi Magnumpci!! 
Here are the new language lines for the new version of Start11 v2.0 beta 1.02. I have already finished the translation into Spanish.

Best Wishes!!
Francis




;2.0 Beta 1.02
"The classic Windows 7 menu."="Menú clásico de Windows 7."
"An updated Windows 7 style."="Menú Windows 7 actualizado."
"Windows 10 style with improvements."="Estilo Windows 10 con mejoras."
"Windows 11 style with improvements."="Estilo Windows 11 con mejoras."
"Windows App style"="Estilo Windows Aplicaciónes"
"Windows Pro style"="Estilo Windows Pro"
"Windows Launcher style"="Estilo Windows Launcher"
"An updated Windows 11 style."="Menú Windows 11 actualizado."
"Modern style with icons"="Estilo Moderno con iconos"
"Grid Modern style with icons"="Estilo Moderno cuadrícula con iconos"
"Change start menu colours"="Cambiar colores del menú de inicio"
"This is always enabled when using Mica background mode."="Siempre está activado cuando se usa el modo de fondo Mica"
"Selected item accent colour"="Acento de color del elemento seleccionado"
"Override the default colour for item selection."="Sobreescribe el color predeterminado para la selección de elementos."
"Adjust menu transparency effects"="Ajuste de efecto de transparencia del menú"
"Use Mica effect on menu background"="Usar efecto Mica en el fondo del menú"
"Use this to use the Mica effect rather than standard blur on the menu."="Usa esto para utilizar el efecto Mica en lugar del desenfoque estándar en el menú."
;TRANSLATION DONT WORK "Menu background transparency level"="Nivel de transparencia del fondo del menú"
"Adjust"="Ajustar"
"Fonts"="Fuentes"
"Control what fonts Start11 should use."="Controla qué fuentes debe utilizar Start11."
;TRANSLATION DONT WORK "Item font :"=""
;TRANSLATION DONT WORK "Header font :"=""
;TRANSLATION DONT WORK "Menu font :"=""
"80% size"="80% del tamaño original"
"90% size"="90% del tamaño original"
"100% size"="100% del tamaño original"
"110% size"="110% del tamaño original"
"120% size"="120% del tamaño original"
"130% size"="130% del tamaño original"
"140% size"="140% del tamaño original"
"** Default font **"="** Fuente predeterminada **"
"Adjust global menu scaling"="Ajustar la escala global del menú"
"Taskbar pins"="Anclar a barra de tareas"
"Start11 allows you to pin documents or folders to your taskbar for easy access"="Start11 te permite anclar archivos o carpetas a tu barra de tareas para un fácil acceso"
"Pin file"="Anclar archivo"
"Create a pin for a document on your taskbar."="Anclar un archivo a tu barra de tareas."
"Pin folder"="Anclar carpeta"
"Create a pin for a folder on your taskbar."="Anclar una carpeta a tu barra de tareas."
"Pin folder menu"="Anclar una carpeta menú"
"Create a pin for a folder popup menu on your taskbar."="Anclar una carpeta con menú desplegable a la barra de tareas."
;TRANSLATION DONT WORK "Pinned items can be unpinned from the taskbar using the right click menu, and their icon can be changed using the properties menu."="Los objetos anclados pueden desanclarse de la barra de tareas usando el menú derecho del ratón, y cambiar sus iconos usando el menú de propiedades."
"Folder menu options"="Opciones carpeta menú"
"Show folders as submenus"="Mostrar carpetas como submenus"
"Manage filtered folders"="Administrar carpetas filtradas"
"Please select a folder to pin"="Selecciona una carpeta para anclar"
"Are you sure you want to pin this?"="¿Estás seguro de que quieres anclar esto?"
"Are you sure you want to pin this to the taskbar?"="¿Estás seguro de que quieres anclar esto a la barra de tareas?"
"** You can unpin it from the right click menu and change icon on the shortcut properties **"="** Puedes desanclarlo con el menú derecho del ratón y cambiar el icono en las propiedades del acceso directo **"
"Browse for folder..."="Selecciona carpeta"
"Suggested apps"="Aplicaciones recomendadas"
"Quick launch"="Acceso rápido"
"Custom filtered folders"="Carpetas personales filtradas"
"No filtered folders found"="No se encuentran carpetas"
"Filtered folders"="Carpetas filtradas"
"Create a custom filtered folder by giving it a name and picking a folder and any rules which should be applied."="Crea una carpeta personalizada con filtros dando un nombre y seleccionando una carpeta y las reglas que deben aplicarse."
;TRANSLATION DONT WORK "Pick an item from above and configure it using the buttons below."="Elige un elemento de los anteriores y configúralo utilizando los botones de abajo."
"Pick target folder..."="Carpeta de destino..."
"Inclusion regex rule"="Regla de inclusión"
"Exclusion regex rule"="Regla de exclusión"
"Enter name"="Escribe un nombre"
"Please enter a unique name for this filtered folder"="Escribe un nombre único para el filtro de esta carpeta"
"Enter filter"="Ingresa filtro"
"RegEx is used on the filename and you can also just enter png for all png files, jpg for all jpg files etc."="Las expresiones regulares (RegEx) se utilizan en el nombre de archivo, y también puedes simplemente ingresar 'png' para todos los archivos PNG, 'jpg' para todos los archivos JPG, etc."
"Learn more about RegEx"="Lee más sobre RegEx"
"Cancel"="Cancelar"
"Are you sure?"="¿Estás seguro?"
;TRANSLATION DONT WORK "Do you want to delete custom filtered folder ?"="¿Estás seguro que deseas eliminar el filtro de la carpeta?"
"< Access denied >"="< Acceso denegado >"
"Show results from Everything instead of Start11 file searches"="Mostrar resultados de Everything en lugar de las búsquedas de archivos de Start11"
"If you have the Everything search tool installed this will be used for non app searches."="Si tienes instalada la herramienta de búsqueda Everything, esta se utilizará para las búsquedas que no sean de aplicaciones."
"Open the Start11 menu on the screen the mouse pointer is on"="Abre el menú de Start11 en la pantalla en la que se encuentra el puntero del ratón"
"Settings backup"="Guardar ajustes"
;TRANSLATION DONT WORK "Backup or restore your Start11 settings"=""
;TRANSLATION DONT WORK "Use the options below to backup or restore your settings."=""
"Backup settings to a file"="Guardar ajustes en un archivo"
"Pick a file to save your settings to."="Selecciona un archivo para guardar tus ajustes."
"Restore settings"="Restaurar ajustes"
"Pick a file to restore your settings from."="Selecciona un archivo para restaurar tus ajustes."
"Pinned shortcuts for the Windows /Modern style menus and pinned taskbar items are not backed up or restored as these are Windows data rather than Start11."=""
;TRANSLATION DONT WORK "Reset Start11 settings"="Restablecer ajustes de Start11"
"Reset settings"="Restablecer ajustes"
"Reset all your Start11 settings back to defaults."="Restablecer por defecto todos los ajustes de Start11."
"Clear information"="Eliminar información"
"Clear all Start11 information for suggested apps and search history."="Eliminar toda la información de Start11 para aplicaciones sugeridas e historial de búsqueda."
"Are you sure you want to reset all your settings back to default?"="¿Estás seguro que deseas restablecer todos tus ajustes a los ajustes por defecto?"
"Are you sure you want to clear your run and search history?"="¿Estás seguro que deseas eliminar el historial de ejecuciones y búsqueda?"
"Manage custom filtered folders"="Administrar carpetas personalizadas con filtros"
"Start11 can show groups or tabs based on a folder with a filter applied to them.  Configure them from here."="Start11 puede mostrar grupos o pestañas basados en una carpeta con un filtro aplicado. Configúralos desde aquí"
"Synchronize pinned items from the OS menu with Start11 (sign out required)"="Sincronizar elementos anclados desde el menú del sistema operativo con Start11 (se requiere cerrar la sesión)"
"Use Windows 11 style folders"="Usar estilo de carpetas de Windows 11"
"Use Windows 10 style folders"="Usar estilo de carpetas de Windows 11"
"Hide username and icon from menu"="Ocultar nombre de usuario e iconos del menú"
"Reduce the size of the right hand side"="Reducir tamaño del lado derecho."
"Add group"="Añadir grupo"
"Add a folder as group"="Añadir carpeta como grupo"
"Tab"="Pestaña"
"Add page"="Añadir página"
"Add folder..."="Añadir carpeta..."
"Add apps folder"="Añadir carpeta de programas"
"Add Steam® folder"="Añadir carpeta de Steam®"
"Add Suggested apps folder"="Añadir carpeta de aplicaciones recomendadas"
"Add Recent searches folder"="Añadir carpeta de búsquedas recientes"
"Tip : Add to this list with custom filtered folders from the Start11 config window"="Consejo : Agrega a esta lista con carpetas personalizadas con filtros desde la ventana de configuración de Start11"
"Add blank tab"="Añadir pestaña en blanco"
"Add a folder as tab"="Añadir carpeta como pestaña"
"Pin installed application..."="Anclar aplicación instalada..."
"Appearance"="Apariencia"
"Background texture"="Textura de fondo"
"Icon tinting"="Tinte de icono"
"Enable text shadows"="Activar sombras de texto"
"Invert text colour"="Invertir color del texto"
"Icon style"="Estilo de iconos"
"Group row count"="Filas en el grupo"
"Save settings as default for new groups"="Guardar configuración para nuevos grupos"
"Move"="Mover"
"Launch entire group"="Ejecutar grupo completo"
"Tip : These textures will be recoloured based on your group colour"="Consejo : Estas texturas se recolorearán según el color de tu grupo"
"Pick custom texture..."="Seleccionar textura personalizada..."
"Draw icon on the left"="Alinear icono a la izquierda"
"Do not auto arrange icons"="No autoorganizar iconos"
"Group settings"="Configuración del grupo"
"Filter group"="Filtro del grupo"
"Reload contents"="Refrescar contenido"
"Move tab right"="Mover pestaña a la derecha"
"Move tab left"="Mover pestaña a la izquierda"
"Detach tab from tab group"="Desvincular pestaña del grupo"
"Show hidden items list"="Mostrar lista de elementos ocultos"
"Clear hidden list"="Limpiar lista de ocultos"
"Stretch to fill"="Estirar para llenar"
"Automatic"="Automático"
"1 row"="1 fila"
;TRANSLATION DONT WORK "2 rows"="2 filas"
;TRANSLATION DONT WORK "3 rows"="3 filas"
;TRANSLATION DONT WORK "4 rows"="4 filas"
;TRANSLATION DONT WORK "5 rows"="5 filas"
;TRANSLATION DONT WORK "6 rows"="6 filas"
;TRANSLATION DONT WORK "7 rows"="7 filas"
;TRANSLATION DONT WORK "8 rows"="8 filas"
;TRANSLATION DONT WORK "9 rows"="9 filas"
;TRANSLATION DONT WORK "10 rows"="10 filas"
;TRANSLATION DONT WORK "12 rows"="12 filas"
;TRANSLATION DONT WORK "14 rows"="14 filas"
;TRANSLATION DONT WORK "16 rows"="16 filas"
;TRANSLATION DONT WORK "18 rows"="18 filas"
;TRANSLATION DONT WORK "20 rows"="20 filas"
"Current tab row count"="Filas actuales en la pestaña"
"Resize all in tab group"="Redimensionar todas las pestañas del grupo"
"Delete group and all tabs"="Eliminar grupo y todas las pestañas"
"Delete this tab"="Borrar pestaña"
"New tab"="Nueva"
"Delete tab"="Borrar pestaña"
"Are you sure you want to delete this tab?"="¿Estás seguro que deseas eliminar esta pestaña?"
"Delete all tabs?"="¿Borrar todas las pestañas?"
"Are you sure you want to delete this group and child tabs?"="¿Estás seguro que deseas eliminar este grupo y sus pestañas?"
"Show folder icon"="Mostrar carpeta de iconos"
"Suggested"="Aplicaciones recomendadas"
"Hide from apps"="Ocultar en las aplicaciones"
"Show suggested apps"="Mostrar aplicaciones recomendadas"
"Show recently installed apps"="Mostrar aplicaciones instaladas recientemente"
"Show flat list without folders"="Mostrar lista sin carpetas"
"Enable all apps searching"="Habilitar búsqueda de todas las aplicaciones"
"Showing hidden items only"="(muestra solo aplicaciones ocultas)"
"To new group"="A nuevo grupo"
"Hide from this group"="Ocultar en éste grupo"
"Pick custom icon..."="Selecciona icono personalizado..."
"Open menu on this page for current desktop"="Abrir menú en esta página para el escritorio actual"
"Group widths"="Ancho de los grupos"
"Reset icon positions"="Restablecer posición de los iconos"
"6 small icons wide"="6 iconos pequeños de ancho"
"8 small icons wide"="8 iconos pequeños de ancho"
"10 small icons wide"="10 iconos pequeños de ancho"
"12 small icons wide"="12 iconos pequeños de ancho"
"14 small icons wide"="14 iconos pequeños de ancho"
"16 small icons wide"="16 iconos pequeños de ancho"
" Type here to search installed apps"="Buscar aplicaciones instaladas"
" Search Apps and files"="Buscar Aplicaciones y archivos"
"You can create filtered folders to show in the start menu or as a taskbar folder menu."="Puedes crear carpetas con filtros para mostrar en el menú de inicio o como un menú de carpetas en la barra de tareas."

**Comment by ghost** [comment]
I am translating Fences to Hebrew. How is it possible to check the lng. file in the software?
